src/index.css

```diff
@@ -0,0 +1 @@
+@import-normalize;
```

Comment:
The PR looks nice and clean. Just FYI: You can use [styled-normalize](https://github.com/sergeysova/styled-normalize) alternatively. Then you don't need this `.css` file.

Reply:
@jkettmann thanks. Fixed. I have never used Styled-components before (and only played with React) so all comments appreciated.